Public/New-HTMLTab.ps1

```diff
@@ -90,6 +91,14 @@ function New-HTMLTab {
     if ($TextColor) {
         $StyleText.'color' = ConvertFrom-Color -Color $TextColor
     }
+    # New-HTMLTab - Add text-transform
+    if($TextTransform){
```

Comment:
Removed the if/else statement and left `$StyleText.'text-transform' = "$TextTransform"`